lib/config/config.go

```diff
@@ -50,7 +50,6 @@ var (
 	DefaultDiscoveryServers = append(DefaultDiscoveryServersV4, DefaultDiscoveryServersV6...)
 
 	// DefaultDiscoveryServersIP is used by the usage reporting.
-	// XXX: Detect Android, and use this is we still don't have working DNS?
 	DefaultDiscoveryServersIP = []string{
```

Comment:
Things may be configured to use the IP addresses, and it's neat if we detect that case for the usage reporting? We should remove it at some point though...